annotator/src/main/java/org/cbioportal/annotator/internal/GenomeNexusImpl.java

```diff
@@ -655,7 +655,7 @@ else if(var.equals("-") || var.length() == 0){
          Example SNP   : 2 216809708 216809709 CA T
          Example output: 2:g.216809708_216809709delCAinsT
          */
-        else if (ref.length() > 1) {
+        else if (ref.length() > 1 || var.length() > 1) {
```

Comment:
variants where ref allele len is 1 but alt allele was > 1 were constructed as `chr:g.XXXXREF>ALT` rather than `chr:g.XXXXdelREFinsALT`

Reply:
this check allows either ref or alt allele to be greater than one and constructs an hgvs as `chr:g.XXXXdelREFinsALT`